### PR TITLE
LPS-169382

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://issues.liferay.com/browse/LPS-169382

### CODE REVIEW

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:stop

**Comment by CaiocSiqueira** [comment]
ci:search:nocompile

**Comment by brookedalton** [comment]
ci:test:publications:nocompile

**Comment by brookedalton** [comment]
ci:stop

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase [hunk 1]
```diff
@@ -1167,6 +1167,35 @@ definition {
...
+		PublicationsNavigator.gotoPublish(
+			gotoPublicationsAdmin = "true",
+			publicationName = "Publication Name");
+
+		AssertTextEquals.assertPartialText(
+			locator1 = "Message#SUCCESS",
+			value1 = "No unresolved conflicts, ready to publish.");
+
+		Publications.publishPublication(publicationName = "Publication Name");
+	}
```
Comment (brookedalton) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase:1197]: Hi @CaiocSiqueira, can you add this after {{Publications.publishPublication}}? {{Publications.viewPublicationPublished(publicationName = "Publication Name");}} and afterwards go to the web content admin and assert the WC Folder is not there?

**Comment by brookedalton** [comment]
Hi @CaiocSiqueira can you also move this test to the PublicationsWebContent file? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase (hunk 1 above)
Comment (CaiocSiqueira) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase:1197]: Sure!

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:search:nocompile

**Comment by CaiocSiqueira** [comment]
ci:stop

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:search:nocompile

**Comment by brookedalton** [comment]
ci:test:publications:nocompile

**Comment by brookedalton** [comment]
Continued in https://github.com/brookedalton/liferay-portal/pull/180

**Pull request closed without merging**